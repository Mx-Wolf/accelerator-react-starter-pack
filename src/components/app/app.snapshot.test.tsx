import * as ShallowRenderer from 'react-test-renderer/shallow';
import App from './app';

describe('Component: App', () => {
  it('should render correctly', () => {
    const view = ShallowRenderer.createRenderer();
    view.render(<App />);
    expect(view).toMatchSnapshot();
  });
});
