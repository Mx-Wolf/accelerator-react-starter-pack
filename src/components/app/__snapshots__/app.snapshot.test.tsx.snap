// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should render correctly 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_didScheduleRenderPhaseUpdate": false,
  "_dispatcher": Object {
    "readContext": [Function],
    "useCallback": [Function],
    "useContext": [Function],
    "useDebugValue": [Function],
    "useDeferredValue": [Function],
    "useEffect": [Function],
    "useImperativeHandle": [Function],
    "useLayoutEffect": [Function],
    "useMemo": [Function],
    "useReducer": [Function],
    "useRef": [Function],
    "useResponder": [Function],
    "useState": [Function],
    "useTransition": [Function],
  },
  "_element": <App />,
  "_firstWorkInProgressHook": null,
  "_forcedUpdate": false,
  "_instance": null,
  "_isReRender": false,
  "_newState": null,
  "_numberOfReRenders": 0,
  "_renderPhaseUpdates": null,
  "_rendered": <Switch>
    <Route
      exact={true}
      path="/"
    >
      <MainScreen />
    </Route>
    <Route
      exact={true}
      path="/guitars"
    >
      <MainScreen />
    </Route>
    <Route
      exact={true}
      path="/guitars/:id"
    >
      <GuitarProductScreen />
    </Route>
    <Route
      exact={true}
      path="/cart"
    >
      <CartScreen />
    </Route>
    <Route>
      <NotFoundPage />
    </Route>
  </Switch>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
  "_workInProgressHook": null,
}
`;
